assets/sass/patterns/organisms/content-header.scss

```diff
@@ -501,7 +501,7 @@ $iconlist: cc oa;
 }
 
 .content-header__author_link_highlight {
-  padding: 0;
+  padding: 12px 0 0;
```

Comment:
Need to remove this padding at wider viewport sizes to prevent this extra vertical space:

![Screenshot 2020-01-28 09 30 24](https://user-images.githubusercontent.com/2893480/73251487-f80ef200-41b0-11ea-87e0-23d83ccc51fd.png)